app/models/concerns/background_deletion.rb

```diff
@@ -5,32 +5,17 @@ module BackgroundDeletion
 
   included do
     class_attribute :background_deletion, default: [], instance_writer: false
+    class_attribute :background_deletion_method, default: :destroy!, instance_writer: false
```

Comment:
Looks like the original default method was `destroy`, while now it's `destroy!`. I guess that was a deliberate choice?.. 
I'm wondering what are the potential consequences though. I mean, probably it's better to just abort the deletion if there is an issue, rather than let some intermediate deletes silently fail... But just wanted to ask anyway :)

Reply:
See my original comment. Originally the default was `destroy` for "by association" and `destroy!` for the main object.

See https://github.com/3scale/porta/blob/1cb77853cb82d3b66d3012137d2b621bb3e24643/app/workers/delete_plain_object_worker.rb#L49-L53
https://github.com/3scale/porta/blob/1cb77853cb82d3b66d3012137d2b621bb3e24643/app/workers/delete_plain_object_worker.rb#L40

No reason I see to avoid bang. I hope testing in stg will reveal any issues.